cloud/releases/cloud-release-notes.md

```diff
@@ -3,6 +3,60 @@ id: cloud-release-notes
 title: Centreon Cloud release notes
 ---
 
+## April 11, 2024
+
+### centreon-web 24.05.1
+
+- [ACL] Added listing of access management rules.
+- [API] Added an endpoint to create Broker outputs.
+- [API] Added an endpoint to PATCH (partial update) openid provider.
+- [API] Added an endpoint to handle fileformat associated to Stream Connector.
+- [API] Update endpoint to handle Status Grid "condensed" view.
+- [API] Added new endpoint to duplicate dashboard.
+- [API] Added API endpoints to return the complete hierarchy for a given BA.
+- [Cloud migration] Added BA in MAP script.
+- [Configuration] Improved service groups configuration.
+- [Configuration] Improved host groups configuration.
+- [Configuration] Updated host configuration form to link host to at least one hostgroup.
+- [Dashboards] Added duplicate feature.
+- [Dashboards] Added new "Status Chart" widget.
+- [Edition] Added a warning modal preventing user from quitting his dashboard without saving it first.
+- [Metrics Graph widget] Added information tooltip when hovering an element on the graph.
+- [Status Grid widget] Added new "condensed" view.
+- [Widgets] Added warning message when no resource is matching the filters selected.
+- [Widgets] Optimized widget rendering.
+- [UX] Made the dashboards library view as list the default one.
+- [UX] Removed "Beta" mention in menus.
+
+###  centreon-gorgone 24.05.1
+
+Enhancement
+
+- [Gorgone] Moved gorgone whitelists in a dedicated file.
+
+### centreon-map 24.05.1
+
+Enhancement
+
+- [Editor] Removed non available features for "Links".
+- [Editor] Removed the possibility to edit link color for status link and metric link.
+- [Protobuf] Implemented "pb_instance_configuration" event.
+
+
+### centreon-it-edition-extensions 24.05.1
+
+Feature
+
+- [CEIP] Added usage statistics for Playlists.
+- [Playlist] New endpoint PATCH for playlists update.
+
+### centreon-bam 24.05.1
+
+Feature
+
+- [BAM] Added BA Diagram widget to Dashboards.
+
+
```

Comment:
### Resource Access Management

Resource Access Management allows administrators to assign specific access permissions to each user or user group. This enables control over which users can access particular monitored resources. An intuitive interface helps manage these permissions, while audit logs provide a record of access activities. This feature supports organizations in maintaining security and organizational efficiency by allowing tailored access control.

### Centreon Dashboards

Centreon Dashboards is now Generaly Available!
In addition to our stabilization works, we have added several features like dashboard duplication, or a new mechanism to prevent exiting edition mode with unsaved changes. 
We have also improved the look and feel of existing widget, in particular graph legends and tooltips and enriched our widget library with three new widgets: Status Chart, Status Grid and Business Activity Diagram.

### Stream Conectors

It is now possible to forward data collected by Centreon to third party systems using our Stream Connectors library.
Please contact the xxx (support? Sales?) team to set up such integrations.


### Pollers

You can now choose amongst all Operating Systems supported by Centreon to install your poller. This includes Alma, RHEL, and Oracle Linux version 8 and 9, Debian version 11 and 12 (from 24.04).